### PR TITLE
Working npm deployment

### PR DESCRIPTION
**Pull request by hpractv**
# Summary of PR changes

Initial working version deployed to `npm`.

## PR Requirements
- [x] For major, minor, or breaking changes, at least one of the commit messages contains the appropriate `+semver:` keywords.
  - See the *Incrementing the Version* section of the repository's README.md for more details.
- [x] The action code does not contain sensitive information.

*NOTE: If the repo's workflow could not automatically update the `README.md`, it should be updated manually with the next version.  For javascript actions, if the repo's workflow could not automatically recompile the action it should also be updated manually as part of the PR.*

### CODE REVIEW

File: src/components/EnvironmentDeployments/EnvironmentDeployments.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+          </div>
+        }
+        actions={[
+          {
+            icon: () => <GitHubIcon />,
+            tooltip: 'Environment Deployments',
+            isFreeAction: true,
+            onClick: () => {
+              window.open(
+                `https://github.com/${props.projectSlug}/deployments/${props.environment}`,
```
Comment (mike-schenk) [inline src/components/EnvironmentDeployments/EnvironmentDeployments.tsx:71]: _This can be fixed in a later PR._ The generated GitHub URL doesn't work when `props.environment` is `"Any"`. We also ought to see if there's a standard way to have the domain part come from configuration... similar to how the new API in this plugin doesn't have "github.com" anywhere in its source.

File: app-config.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  #   keys:
+  #     - secret: ${BACKEND_SECRET}
+  baseUrl: http://localhost:7007
+  listen:
+    port: 7007
+    # Uncomment the following host directive to bind to all IPv4 interfaces and
+    # not just the baseUrl hostname.
+    # host: 0.0.0.0
+  csp:
+    frame-src: ['miro.com', 'web.powerva.microsoft.com']
```
Comment (mike-schenk) [inline app-config.yaml]: This app-config.yaml is just for running the plugin "stand alone", so it shouldn't have these unneeded CSP frame sources.

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  # - Prefer team owners over individuals.
+  # - Paths are case sensitive.
+  # - Order is important.  The last matching pattern takes the most precedence.
+
+  # These owners will be the default owners for everything in the repo. Unless a later match takes
+  # precedence, these teams will be requested for review when someone opens a pull request.
+  # Draft PRs do not automatically request reviews until they are marked as Ready for Review.
+  # If the repo's Branch Protection Rule requires code owner reviews, one of these team members
+  # must sign off before a PR can be merged.
+  * @im-open/swat
```
Comment (mike-schenk) [inline .github/CODEOWNERS:15]: GitHub is saying this team isn't valid here.

**Review by mike-schenk — CHANGES_REQUESTED** [review]

File: app-config.yaml (hunk 2 above)
Comment (hpractv) [reply app-config.yaml]: 👍🏽

File: .github/CODEOWNERS (hunk 3 above)
Comment (hpractv) [reply .github/CODEOWNERS:15]: I just had to add swat as contributors to the repo.

File: src/components/EnvironmentDeployments/EnvironmentDeployments.tsx (hunk 1 above)
Comment (hpractv) [reply src/components/EnvironmentDeployments/EnvironmentDeployments.tsx:71]: I'll try to come up with a better way to do this.  This link won't show up for a environemnt of "Any", but I see what you mean about this part.

**Review by mike-schenk — APPROVED** [review]

File: src/components/EnvironmentDeployments/EnvironmentDeployments.tsx (hunk 1 above)
Comment (mike-schenk) [reply src/components/EnvironmentDeployments/EnvironmentDeployments.tsx:71]: Well... [I saw the link](https://techhub.uat.mktp.io/catalog/default/component/applications-and-policies-mfe/deployments) for an environment of "Any" which is why I reported it.

Comment (hpractv) [reply src/components/EnvironmentDeployments/EnvironmentDeployments.tsx:71]: Oh wait.  I just realized this was for the link at the top right.  I'll clean this up.

**Pull request merged**